frontend/src/pages/invite/ui/inviteMessage.ts

```diff
@@ -0,0 +1,101 @@
+import { msg } from "@lit/localize";
+import { html, type TemplateResult } from "lit";
+import { when } from "lit/directives/when.js";
+
+import type { UserOrgInviteInfo } from "@/types/user";
+
+export const renderInviteMessage = (
+  inviteInfo: void | UserOrgInviteInfo,
+  { isExistingUser, isOrgMember } = {
+    isExistingUser: false,
+    isOrgMember: false,
+  },
+) => {
+  if (!inviteInfo) return;
+
+  let message: string | TemplateResult = "";
+
+  if (inviteInfo.firstOrgAdmin) {
+    if (isExistingUser) {
+      message = msg("Set up your organization to start web archiving.");
```

Comment:
Would change to "crawling", the verb we use for running a "crawl workflow".  Also fits in with some of our other marketing copy: "Automated browser-based crawling at scale" etc.

"to start archiving" also acceptable?  We do use "archive" as a verb!

```suggestion
      message = msg("Set up your organization to start crawling.");
```